<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>login</class>
 <widget class="QDialog" name="login">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>250</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>202</y>
     <width>211</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>112</y>
     <width>54</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Password:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_name">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>70</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Name  :</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cb_host">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>150</y>
     <width>150</width>
     <height>24</height>
    </rect>
   </property>
   <property name="editable">
    <bool>true</bool>
   </property>
   <item>
    <property name="text">
     <string notr="true">192.168.2.168</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>150</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Host :</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cb_name">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>70</y>
     <width>150</width>
     <height>24</height>
    </rect>
   </property>
   <property name="editable">
    <bool>true</bool>
   </property>
   <item>
    <property name="text">
     <string notr="true">debug</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>hell</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>10</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>====    Hell Jabber Client    ====</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_pwd">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>110</y>
     <width>150</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>debug</string>
   </property>
   <property name="maxLength">
    <number>64</number>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="placeholderText">
    <string notr="true"/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>login</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>login</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
